CHANGELOG.md

```diff
@@ -101,6 +101,7 @@ END_UNRELEASED_TEMPLATE
   `# gazelle:python_resolve_sibling_imports true`
 * (pypi) Show overridden index URL of packages when downloading metadata have failed.
   ([#2985](https://github.com/bazel-contrib/rules_python/issues/2985)).
+* (toolchains) `local_runtime_repo` better handles variants in MacOS and Windows.
```

Comment:
You could also describe that `linux` got improved as well.

Reply:
I expanded the changelog